server/main.lua

```diff
@@ -295,9 +295,7 @@ lib.callback.register('qbx_admin:server:spawnVehicle', function(source, model)
         warp = true,
     })
 
-    local plate = qbx.getVehiclePlate(NetworkGetEntityFromNetworkId(netId))
-
-    exports.qbx_vehiclekeys:GiveKeys(source, plate)
+    exports.qbx_vehiclekeys:GiveKeys(source, NetworkGetEntityFromNetworkId(netId))
```

Comment:
This is unneeded, I believe, as the `spawnVehicle` function returns both the entity and the netId if I remember correctly.

Reply:
Thank you. I'm reviewing the code, a lot of it is still using old functions, I'm readapting it here. I just saw that a function is called to add a key to the client, but it wasn't working for some reason, the admin script itself is using a lot of functions internally (qb-*).